c2rust-transpile/tests/snapshots/pause.rs

```diff
@@ -0,0 +1,18 @@
+#![allow(
+    dead_code,
+    mutable_transmutes,
+    non_camel_case_types,
+    non_snake_case,
+    non_upper_case_globals,
+    unused_assignments,
+    unused_mut
+)]
+#![feature(stdsimd)]
```

Comment:
So `fn import_simd_function` unconditionally adds `#![feature(stdsimd)]` even for stable functions like `core::arch::x86_64::_mm_pause`, which isn't ideal, but should probably be fixed separately.